packages/venia-concept/src/components/SearchBar/getLocation.js

```diff
@@ -0,0 +1,13 @@
+export default (searchValue, categoryId) => {
+    // start with the current uri
+    const uri = new URL('/search.html', window.location);
```

Comment:
At some point in the future, we'll want to power this via a build-time query against store configuration for its preferred search URL. Can you add a TODO for that?

Reply:
Comment added. 👍